global.json

```diff
@@ -1,15 +1,15 @@
 {
   "sdk": {
-    "version": "5.0.100-preview.7.20366.6",
+    "version": "5.0.100-preview.8.20417.9",
     "allowPrerelease": true,
     "rollForward": "major"
   },
   "tools": {
-    "dotnet": "5.0.100-preview.7.20366.6",
+    "dotnet": "5.0.100-preview.8.20417.9",
     "vs": {
-      "version": "16.6"
+      "version": "16.9"
```

Comment:
This seemed to work for the sdk to force a minimum of 16.8p2, so I'm giving it a try: https://github.com/dotnet/sdk/blob/master/global.json#L9-L10

Reply:
this does not work on the public previews. This will need to be 16.8

I get this locally for 16.8 P2:
```
(NETCORE_ENGINEERING_TELEMETRY=InitializeToolset) Developer Command Prompt for VS 16.0 is not recent enough. Please upgrade to 16.9 or build from a plain CMD window
```